sys/linux/socket_inet.txt

```diff
@@ -204,3 +205,31 @@ arpreq_in {
 	arp_netmask	sockaddr_in
 	arp_dev		devname
 }
+
+# sendmsg for ip_cmsg_send
+
+msghdr_inet {
+	msg_name	ptr[in, sockaddr_in, opt]
+	msg_namelen	len[msg_name, int32]
+	msg_iov		ptr[in, array[iovec_in]]
+	msg_iovlen	len[msg_iov, intptr]
+	msg_control	ptr[in, array[cmsghdr_inet], opt]
+	msg_controllen	bytesize[msg_control, intptr]
+	msg_flags	const[0, int32]
+}
+
+mmsghdr_inet {
+	msg_hdr	msghdr_inet
+	msg_len	const[0, int32]
+}
+
+cmsghdr_inet [
+	ip_retopts	cmsghdr_t[SOL_IP, IP_RETOPTS, string]
```

Comment:
Are these the same options as in `ipv4_options` from `vnet.txt`? If so, use `ipv4_options` instead of `string`.